storage-node/packages/runtime-api/index.js

```diff
@@ -31,6 +31,7 @@ const { DiscoveryApi } = require('@joystream/storage-runtime-api/discovery')
 const { SystemApi } = require('@joystream/storage-runtime-api/system')
 const AsyncLock = require('async-lock')
 const Promise = require('bluebird')
+const { sleep } = require('@joystream/storage-utils/sleep')
```

Comment:
the storage-utils packages doesn't have an index.js .. so importing like this, but we should fix in future.